node/core/provisioner/Cargo.toml

```diff
@@ -10,9 +10,12 @@ futures = "0.3.17"
 tracing = "0.1.29"
 thiserror = "1.0.30"
 polkadot-primitives = { path = "../../../primitives" }
+polkadot-node-primitives = { path = "../../primitives" }
 polkadot-node-subsystem = { path = "../../subsystem" }
 polkadot-node-subsystem-util = { path = "../../subsystem-util" }
 futures-timer = "3.0.2"
+rand = "0.8.4"
+thousands = "0.2.0"
```

Comment:
I looked quickly throughout the polkadot repo and I haven't seen any other uses of this dependency in the tree. I am not entirely convinced that the use-case warrants adding another dep.

Reply:
Fair point, it just really helps with readability of large numbers.

Reply:
Yeah, understandable. I hope we won't have to deal with that too often though )

Reply:
Just had to debug some weight's lately, and there it really pays off

Reply:
Removed again.